.github/workflows/releasedr.yaml

```diff
@@ -62,9 +65,21 @@ jobs:
         run: |
           sleep 30
 
+      - name: "Get current umbrella chart version"
+        id: parent-chart
+        uses: mikefarah/yq@v4.33.3
+        with:
+          cmd: echo "version=$(yq '.version' charts/datarepo/Chart.yaml)" >> $GITHUB_OUTPUT
+
       - name: "Create new umbrella chart"
         uses: broadinstitute/datarepo-umbrella-release-action@0.3.0
 
+      - name: "Get new umbrella chart version"
```

Comment:
I don't understand the difference between this step and ```Get current umbrella chart version```

Reply:
Thanks for the callout. The datarepo-umbrella-release-action [here](https://github.com/broadinstitute/datarepo-umbrella-release-action/blob/master/src/main.sh) 
does a bunch of stuff, including bumping the checked-out code's chart version number. so the `new-version` step reads the same line as `parent-version` but expects it to return the new version.

I've added a big comment to outline the expectations of the step, so someone who isn't familiar with the `datarepo-umbrella-release-action` script has an idea of what's going on.

Reply:
also, the test run copies the code [here](https://github.com/broadinstitute/datarepo-umbrella-release-action/blob/89c87e50c3c978425ed2e7733e2bbba60a33c4ed/src/main.sh#L22) in the github action to pretend the version is being bumped.